src/app/components/status-indicator/index.html

```diff
@@ -0,0 +1,112 @@
+<stache
+  pageTitle="Status indicator"
+  windowTitle="Status indicator"
+  layout="container"
+	showTableOfContents="true">
+
+<p>
+	The status indicator classes create styled text to draw attention to data that needs to be addressed. A status is "a state or condition at a particular time". Statuses fall into four categories: success, warning, danger, and info. Examples include late, submitted, awaiting approval, etc.
+</p>
+
+<stache-page-anchor>
+	Status indicator classes
+</stache-page-anchor>
+
+  <div class="definition-table definition-table-wide">
+    <div class="definition-item">
+      <div class="sky-text-info">Info</div>
+    </div>
+    <div class="definition-props">
+      <div class="definition-usage">
+        <div class="sky-field-label">Usage</div>
+        <div>This class prepends a blue info icon to provide a compact way to indicate a "info" status.</div>
+      </div>
+      <div class="definition-code">
+        <div class="sky-field-label">Code</div>
+        <stache-code>&lt;div class=&quot;sky-text-info&quot;&gt;Field label&lt;/div&gt;</stache-code>
+      </div>
+      <div class="definition-styling">
+        <div class="sky-field-label">Styling</div>
+        <div>::before fa-exclamation-circle #00b4f1</div>
+      </div>
+    </div>
+  </div>
+	
+	<div class="definition-table definition-table-wide">
+    <div class="definition-item">
+      <div class="sky-text-success">Success</div>
+    </div>
+    <div class="definition-props">
+      <div class="definition-usage">
+        <div class="sky-field-label">Usage</div>
+        <div>This class prepends a green checkmark icon to provide a compact way to indicate a "success" status.</div>
+      </div>
+      <div class="definition-code">
+        <div class="sky-field-label">Code</div>
+        <stache-code>&lt;div class=&quot;sky-text-success&quot;&gt;Field label&lt;/div&gt;</stache-code>
+      </div>
+      <div class="definition-styling">
+        <div class="sky-field-label">Styling</div>
+        <div>::before fa-check #71bf43</div>
+      </div>
+    </div>
+  </div>
+
+  <div class="definition-table definition-table-wide">
+    <div class="definition-item">
+      <div class="sky-text-warning">Warning</div>
+    </div>
+    <div class="definition-props">
+      <div class="definition-usage">
+        <div class="sky-field-label">Usage</div>
+        <div>This class prepends a yellow warning icon to provide a compact way to indicate a "warning" status.</div>
```

Comment:
We say "yellow," but it looks orange to me. We also say "yellow" at https://developer.blackbaud.com/skyux/design/styles/typography, but is that correct?

Reply:
It's pretty much halfway between yellow and orange, but we went with the yellow description to reflect the stoplight metaphor it's based on.

Reply:
OK. Thanks.